src/pages/chat/Chat.tsx

```diff
@@ -141,6 +153,88 @@ export default function Chat() {
   const [previousPath, setPreviousPath] = useState<string>("");
   const navigate = useKeepQueryParamsNavigate();
   const { isXXS, isXS, isS } = useBreakpoints();
+  const { showModal, modalTypes } = useModal();
+  const { bosonXmtp } = useChatContext();
+  const notifiConfig = getNotifiConfig();
+
+  // TODO: ideally if notifiConfig is null, do not call useNotifiClient()
+
+  const notifiClient = useNotifiClient({
+    dappAddress: notifiConfig?.dappId || "",
+    env: "Development",
+    walletPublicKey: address as string,
+    walletBlockchain: notifiConfig?.chain || "ETHEREUM"
+  });
+  const fetchNotifiData = (data: NotifiClientData | null) => {
+    const xmtpAlert = data?.alerts?.find(
+      (alert: { filter?: { filterType?: string } }) =>
```

Comment:
```suggestion
      (alert) =>
```
no need to remove the original autocomplete

Reply:
ok